ipl/procs/equiv.icn

```diff
@@ -76,12 +86,12 @@ procedure equiv(x1,x2,done)		#: compare values for equivalence
    image(x1) ? (code := (="record" | type(x1)))
 
    case code of {
-      "list" | "record": 
+      "list" | "record":
          every i := 1 to *x1 do
             if not equiv(x1[i],x2[i],done) then fail
       "table": if not equiv(sort(x1,3),sort(x2,3),done) then fail
       "set":   if not equiv(sort(x1),sort(x2),done) then fail
-      default: fail			# Vaues of other types are different. 
+      default: fail			# Vaues of other types are different.
```

Comment:
Another typo: Vaues

Reply:
I have added another commit to update the spelling in the comment